core/server/helpers/get.js

```diff
@@ -17,7 +17,7 @@ var proxy = require('./proxy'),
     get;
 
 // Endpoints that the helper is able to access
-resources = ['posts', 'tags', 'users', 'pages'];
+resources = ['posts', 'tags', 'users', 'pages', 'authors'];
```

Comment:
Because we have no diff between content and admin API controllers, you could still do `{{#get "users}}`.
Fix

Reply:
This is fine for now. We just need to ensure that V2 uses the authors controller. Can be achieved with aliases.